activemodel/test/cases/validations/numericality_validation_test.rb

```diff
@@ -233,6 +233,31 @@ def test_validates_numericality_with_symbol
     Topic.remove_method :max_approved
   end
 
+  def test_validates_numericality_with_symbol_and_nil_value
+    Topic.define_method(:max_approved) { nil }
+    Topic.validates_numericality_of :approved, less_than_or_equal_to: :max_approved
+
+    valid!([4])
```

Comment:
Looks this spec (`4 <= nil` is valid) is too application specific to me.

Reply:
That's probably true – I cannot come up with a different example rather than a pair of `min` and `max` fields, but `ArgumentError` seemed confusing to me and a workaround to fix that (`greater_than: ->(filter) { filter.min_price.to_i }`) looks a bit wordy

Reply:
@kamipo how can I make this spec look better?